src/Components/IngredientCard.module.css

```diff
@@ -0,0 +1,19 @@
+.ingredientName {
+  font-weight: bold;
+  border: 1px solid black;
+  border-radius: 10px;
+  display: flex;
+  justify-content: center;
+  padding: 2px;
+  text-align: center;
+}
+
+.imgStyle {
+  border-radius: 15px;
+}
+
+.ingredientName:hover,
+.imgStyle:hover {
+  cursor: pointer;
+  border: 2px solid black;
```

Comment:
Be careful making changes to the border like this. The border is a part of the measurement of space an element takes up on the screen. Because of this, the image takes up 4px x 4px more space which pushes other page elements around. To get an outline of the same nature that will not cause a layout shift, use `    outline: 2px solid black;`